src/pages/contributing/get-started.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Get started
+description:
+  The Carbon Design System is made possible through a vibrant community of
+  designers and developers. Anyone can contribute code, design, documentation,
+  and ideas. Here's how you can contribute, too!
+---
+
+<PageDescription>
+
+The Carbon Design System is made possible through a vibrant community of
+designers and developers. Anyone can contribute code, design, and documentation.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Get started</AnchorLink>
+<AnchorLink>Contribution gallery</AnchorLink>
+<AnchorLink>Carbon champions</AnchorLink>
+<AnchorLink>Types of contribution</AnchorLink>
+<AnchorLink>The process</AnchorLink>
+
+</AnchorLinks>
+
+## Get started
+
+If you're interested in getting involved but don't know where to start, welcome!
+Designers and developers like you help to make Carbon great, and so we're glad
```

Comment:
> Designers and developers like you help to make Carbon great, and so we’re glad you’re here.

Do you mean the above^ or do you mean "...and we're so glad you're here." Could be either but just wanted to double check with you.

Reply:
Good catch! I meant the later.